domain.yml

```diff
@@ -1,753 +1,821 @@
   utter_nicetomeeyou:
-    - text: "Thank you. It is a pleasure to meet you as well!"
-    - text: "It is nice to meet you too!"
-    - text: "Pleased to meet you too!"
-    - text: "Likewise!"
+  - text: Thank you. It is a pleasure to meet you as well!
+  - text: It is nice to meet you too!
+  - text: Pleased to meet you too!
+  - text: Likewise!
   utter_telljoke:
-    - text: "Why are eggs not very much into jokes? - Because they could crack up."
-    - text: "Do you know a tree’s favorite drink? - Root beer!"
-    - text: "Why do the French like to eat snails so much? - They can’t stand fast food."
+  - text: Why are eggs not very much into jokes? - Because they could crack up.
+  - text: Do you know a tree’s favorite drink? - Root beer!
+  - text: Why do the French like to eat snails so much? - They can’t stand fast food.
   utter_ask_whatismyname:
-    - text: "It's probably the one that your parents chose for you."
+  - text: It's probably the one that your parents chose for you.
   utter_ask_howbuilt:
-    - text: "I was built with a lot of love and patience."
+  - text: I was built with a lot of love and patience.
   utter_ask_python_installed:
-    - text: "Do you have Python installed?"
+  - text: Do you have Python installed?
   utter_get_python:
-    - text: "Please install Python first. [Here is a guide](https://realpython.com/installing-python/) how to do so."
-  utter_ask_pip_or_conda:
-    - text: "Do you want to use pip or conda for the installation?"
-  utter_installation_with_pip:
-    - text: "Ok, install the Rasa Stack by executing 'pip install rasa_core' on your command line."
-  utter_installation_with_conda:
-    - text: "Ok, install the Rasa Stack by executing 'conda install rasa_core' on your command line."
+  - text: Please install Python first. [Here is a guide](https://realpython.com/installing-python/)
+      how to do so.
+  utter_run_rasa_init:
+  - text: You can create a new Rasa project by running 'rasa init'. This creates the
+      necessary training and configuration files, and also trains your first model
+      using some example training data.
+  utter_start_rasax:
+  - text: You may run the cell below to start Rasa X. After a few seconds, you will
+      see a button that will take you to its UI.
+  utter_ask_continue_rasa_init:
+  - text: Do you want to go ahead with the steps to create a bot?
   utter_ask_ready_to_build:
-    - text: "Ready to start building a chatbot?"
+  - text: Ready to start building a chatbot?
   utter_ask_describe_problem:
-    - text: "Can you describe the problem you're having?"
+  - text: Can you describe the problem you're having?
   utter_direct_to_forum_for_help:
-    - text: "I can't help you with that 🙁. Please get help in our [forum](https://forum.rasa.com/)."
-  utter_get_starter_pack:
-    - text: "Great, you installed Rasa! If you don't want to start from scratch, you can use our [starter pack](https://github.com/RasaHQ/starter-pack-rasa-stack)."
```

Comment:
We've deprecated the starter packs, all mention of them should be taken out in the stories. `rasa init` handles this

Reply:
That's me removing it from the domain 😄.